unity/PitayaExample/Assets/Pitaya/PitayaBinding.cs

```diff
@@ -62,6 +62,8 @@ public static class PitayaBinding
         private static readonly Dictionary<IntPtr, int> EventHandlersIds = new Dictionary<IntPtr, int>();
         private static PitayaLogLevel _currentLogLevel = PitayaLogLevel.Disable;
         private static bool IsNativeLibInitialized;
+        
+        public static IPitayaQueueDispatcher QueueDispatcher { get; set; } = new NullPitayaQueueDispatcher();
```

Comment:
Shouldn't this be private, since you're passing the dispatcher in the constructor?

Reply:
Yes and no xD.
We pass it in the constructor of `PitayaClient`, which then sets the instance variable within `PitayaBinding` through the public setter. `PitayaBinding` is static, so it seems this is the right approach.

Reply:
Fair enough.